<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConsultationBegin</class>
 <widget class="QDialog" name="ConsultationBegin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Consultation</string>
  </property>
  <widget class="QListWidget" name="FactList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>140</width>
     <height>221</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>140</width>
     <height>310</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="defaultDropAction">
    <enum>Qt::IgnoreAction</enum>
   </property>
   <property name="alternatingRowColors">
    <bool>false</bool>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectItems</enum>
   </property>
   <property name="isWrapping" stdset="0">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="BeginInferenceButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Begin Inference</string>
   </property>
  </widget>
  <widget class="QFrame" name="EditorFrame">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>170</width>
     <height>180</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>170</width>
     <height>180</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>170</width>
     <height>180</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QDoubleSpinBox" name="ConfidenceEditor">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>170</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>170</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>170</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="decimals">
     <number>6</number>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.000001000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="TypeLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Type:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ConfidenceLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Confidence Factor:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ValueEditor">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>170</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>170</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>170</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="TypeIndicator">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>170</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>170</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>170</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ValueLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Value:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveFactValueButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Calibri Light</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Save Fact Value</string>
    </property>
   </widget>
  </widget>
  <zorder>EditorFrame</zorder>
  <zorder>FactList</zorder>
  <zorder>BeginInferenceButton</zorder>
 </widget>
 <tabstops>
  <tabstop>FactList</tabstop>
  <tabstop>TypeIndicator</tabstop>
  <tabstop>ValueEditor</tabstop>
  <tabstop>ConfidenceEditor</tabstop>
  <tabstop>SaveFactValueButton</tabstop>
  <tabstop>BeginInferenceButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>FactList</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>ConsultationBegin</receiver>
   <slot>FactSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SaveFactValueButton</sender>
   <signal>clicked()</signal>
   <receiver>ConsultationBegin</receiver>
   <slot>SaveFactValue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BeginInferenceButton</sender>
   <signal>clicked()</signal>
   <receiver>ConsultationBegin</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>FactSelected()</slot>
  <slot>SaveFactValue()</slot>
 </slots>
</ui>
